docs/language-server/protocol-project-manager.md

```diff
@@ -304,22 +381,377 @@ interface ProjectListSampleResponse {
 
 TBC
 
+## Action Progress Reporting
+
+### `task/started`
+
+Indicates that a long running task has been started.
+
+Currently only used when missing components are being installed to show
+installation progress.
+
+- **Type:** Notification
+- **Direction:** Server -> Client
+- **Connection:** Protocol
+- **Visibility:** Public
+
+#### Parameters
+
+```typescript
+interface TaskStartNotification {
+  taskId: UUID;
+  taskName: String;
```

Comment:
Hmm, I actually haven't thought through what exactly the name should represent.
My initial idea was that it would be some label related to the specific task (i.e. `downloading`, `extracting`), but if the method name is more helpful for you, I think it can be implemented without issue.

Reply:
In this case I think I will rename this field into `relatedOperation` maybe?